tests/unit/test_sds_in_a_box_stack.py

```diff
@@ -4,13 +4,12 @@
 
 import aws_cdk as core
 import aws_cdk.assertions as assertions
-
 from sds_in_a_box.sds_in_a_box_stack import SdsInABoxStack
 
 # example tests. To run these tests, uncomment this file along with the example
 # resource in sds_in_a_box/sds_in_a_box_stack.py
 def test_sqs_queue_created():
-    app = core.App()
+    app = core.App(context={"SDSID": "unit-testing"})
     stack = SdsInABoxStack(app, "sds-in-a-box")
     template = assertions.Template.from_stack(stack)
```

Comment:
I'm not following this test. Are there assertions in here? It seems like it is just creating the stack and template, but not checking against what is in the stack or template...

Name of the test should be updated as well.

Reply:
I actually didn't add the unit test, it comes by default when you set up any new CDK project (including the function name "test_sqs_queue_created") and I just never removed it from the repo.  

This unit test started running after github actions was set up to run the unit tests, so I was just making a small fix here to stop github from complaining about the pull request failing some checks.    

I can either fix the unit tests here, or we could make another ticket for it

Reply:
Sure, we can make another ticket, I was just confused here! We have a lot of test work to do, so I think we can break things down into more specific actions then.